src/utils/encode.ts

```diff
@@ -23,6 +32,16 @@ export function arrayBufferToString(array: ArrayBuffer): string {
  * Convert utf8-string to Uint8Array
  *
  * *[internal usage]*
+ *
+ * @param {string} str The UTF-8 string to convert.
+ * @returns {Uint8Array} The encoded Uint8Array.
+ *
+ * @example
+ * ```typescript
+ * const myString = 'Hi';
+ * const byteArray = utf8ToArray(myString);
+ * console.log(byteArray); // Outputs: Uint8Array {0: 72, 1: 105}
```

Comment:
Have you tested your example?
The result is not a struct, but an array (as the name is explaining) :
```
[ 72, 105 ]
```

Reply:
maybe, but ts-node is reporting :
![image](https://github.com/starknet-io/starknet.js/assets/81040730/d0ecf16a-f1f9-4a68-9409-5167c6e93ec8)
So, Uint8Array is an array... sounds so logical...